backend/internal/adapters/secondary/repositories/database/database.go

```diff
@@ -15,6 +17,8 @@ func NewDatabase(cfg *config.Config) (*sqlx.DB, error) {
 	if err != nil {
 		return nil, fmt.Errorf("error connecting to database: %w", err)
 	}
+	db.MapperFunc(camelToSnake)
```

Comment:
Yup! We should have the domain models be independent of the database. I don't really like struct tags since they entirely bypass the type system, though I suppose they wouldn't be too bad in this particular context.

To confirm, this is so that we can map between structs in the repositories and the database without having to manually specify, right?

Reply:
> To confirm, this is so that we can map between structs in the repositories and the database without having to manually specify, right?

Yep!

Reply:
I think this is fine then, manually specifying struct tags isn't really much better as compared to this solution. Both solutions are quite magical, but this one is less prone to error and enforces that the struct field names correspond to the database column names. Thanks!